### PR TITLE
edits to the docs based on the work currently done

### LINKED ISSUE #96: Update specs to reflect The Current State Of Affairs
See title

### PR DESCRIPTION
**Pull request by shiftkey**
Resolves #96 

image

Anything that's:
- grossly inaccurate?
- in the wiki but should be here?
- missing?

### CODE REVIEW

File: specs/ClientImplementation.md [hunk 1]
```diff
@@ -69,73 +69,75 @@ doesn't actually exist, we'll download the latest full release and set up the
...
+        string SHA1
+        string Filename
+        long Filesize
+        bool IsDelta
+
+## Applying Updates
+
+#### A note about Reactive Extensions
+
+Shimmer uses Reactive Extensions (Rx) heavily as the process necessary to retrieve, download and apply updates is best done asynchronously. If you are using the `Microsoft.Bcl.Async` package (which Shimmer also uses) you can combine the Rx APIs with the TPL async/await keywords, for maximum simplicity.
```
Comment (anaisbetts) [inline specs/ClientImplementation.md]: 78-char word wrap plz

File: specs/ClientImplementation.md [hunk 2]
```diff
@@ -69,73 +69,75 @@ doesn't actually exist, we'll download the latest full release and set up the
...
+
+### Check yourself
+
+First, check the location where your application updates are hosted:
+
+    var updateManager = new UpdateManager(@"C:\Users\brendanforster\Desktop\TestApp",
```
Comment (anaisbetts) [inline specs/ClientImplementation.md]: Wrap this in "```cs" and it'll be syntax highlighted and hot

File: specs/Tools.md [hunk 3]
```diff
@@ -2,12 +2,12 @@
...
 At the end of the day, here's how a developer will use Shimmer:
 
-1. Add the "Shimmer" package to your application
-1. The post-install for Shimmer enables NuGet Package Build (i.e. flips the
-   bits in NuGet.targets and enables NuGet Package Restore)
-1. From the NuGet package console, run `Create-Release` - this builds the
-   world, and you end up with `$SolutionDir/Releases` folder that has both a
-   Shimmer release folder as well as a `Setup.exe`
+1. Add the **Shimmer** package to your application
+1. As part of the install for Shimmer, NuGet Package Build  is enabled in the csproj file
```
Comment (anaisbetts) [inline specs/Tools.md:6]: Fuck yeah that's rad

File: specs/ClientImplementation.md (hunk 2 above)
Comment (shiftkey) [inline specs/ClientImplementation.md]: That's how I had it in the wiki but I felt bad for the non-GH-flavoured-Markdown clients...

But yes, syntax-highlighting FTW

**Pull request merged**